dicom_validator/tests/spec_reader/test_condition_parser.py

```diff
@@ -796,13 +796,17 @@ def disabled_test_ispresent_with_value(self, parser):
             "and the first data point is the same as the last data point."
         )
         assert result.type == ConditionType.MandatoryOrUserDefined
-        assert len(result.or_conditions) == 3
+        assert len(result.or_conditions) == 2
```

Comment:
Right - I also noticed that I already had this diabled test, because I noticed that it doesn't work, but obviously didn't understand the condition at the time.

Reply:
I've added the test to the [previous commit](https://github.com/pydicom/dicom-validator/pull/116/commits/731f1f5b4b7fca9db1d20feee1dbabb1828d091c), and have realized just now that this can be a bit confusing, so sorry for that. 🙂